src/routes/HomePage.jsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { filterCoin, sortByPrice, sortByRank } from '../redux/coins/coinsSlice';
+import rightArrow from '../styles/arrow_circle_right.png';
+
+const HomePage = () => {
+  const { coinsList } = useSelector((store) => store.coins);
+  const dispatch = useDispatch();
+  const itemsPerPage = 10;
+  const [currentPage, setCurrentPage] = useState(1);
+  const indexOfLastItem = currentPage * itemsPerPage;
+  const indexOfFirstItem = indexOfLastItem - itemsPerPage;
+  const currentItems = coinsList.slice(indexOfFirstItem, indexOfLastItem);
+  const totalPages = Math.ceil(coinsList.length / itemsPerPage);
+
+  const handlePageChange = (pageNumber) => {
+    setCurrentPage(pageNumber);
+  };
```

Comment:
- [OPTIONAL] It's one of the project requirements for the Home page to have a list of categories that could be filtered using given parameters, e.g. category name. Please address this. ✍🏻 

![image](https://github.com/Sparktan24/metrics-webapp-cryptocurrency/assets/79058364/9df773cc-6ea6-4601-9468-2317571b9e88)